ui/src/store/modules/oidc.ts

```diff
@@ -0,0 +1,11 @@
+import { vuexOidcCreateStoreModule } from 'vuex-oidc'
+
+export const oidc = vuexOidcCreateStoreModule({
+  // authority: process.env.AUTH_AUTHORITY,
+  // client_id: process.env.AUTH_CLIENT_ID,
```

Comment:
@martinmaillard I could not find how the `process.env` would be injected. Does not work, however `process.env.VUE_APP_API_URL` works. What is this about?

Reply:
I think variables need to start with `VUE_APP_`.

One thing though: using env variables for anything that is environment-specific will force us to have a separate build (thus a separate docker image) for each environment, as there are no "runtime env variables" in the browser. Is this ok? Is this already what we do?